.vscode/settings.json

```diff
@@ -0,0 +1,5 @@
+{
+  "cSpell.words": [
+    "cloudinary"
+  ]
+}
```

Comment:
Please remove this `.vscode/settings.json` file. Put it in `gitignore`

Reply:
changes effected. thanks